README.md

```diff
@@ -1,15 +1,28 @@
 # Restlet Framework
 
-## The leading RESTful web API framework for Java
+## The leading RESTful Web API framework for Java
 
 Thanks to Restlet Framework's powerful routing and filtering capabilities, unified client and server Java API, developers can build secure and scalable RESTful web APIs.
 
-It is available in editions for all major platforms (Java SE/EE, Google AppEngine, OSGi, GWT, Android) and offers numerous extensions to fit the needs of all developers.
+It is available in editions for all major platforms (Java SE/EE, Google App Engine, OSGi, GWT, Android) and offers numerous extensions to fit the needs of all developers.
 
 It is available under the terms of either the Apache Software License 2.0 or the Eclipse Public License 1.0.
 
 http://restlet.com
 
+## Learn more
+
+To learn more about Restlet Framework, please have a look at the following resources:
+
+* [User Guide](http://restlet.com/technical-resources/restlet-framework/guide/)
+* [Tutorials](http://restlet.com/technical-resources/restlet-framework/tutorials/)
+* [JavaDocs](http://restlet.com/technical-resources/restlet-framework/jdocs/)
+* [Change Log](http://restlet.com/technical-resources/restlet-framework/misc/2.3/changes)
```

Comment:
well, to be precise, on master branch we should use this url:
http://restlet.com/technical-resources/restlet-framework/misc/3.0/changes

and you should add another PR for the 2.3 branch